tests/integration/cartography/test_basic.py

```diff
@@ -13,4 +13,4 @@ def test_neo4j_connection():
 def test_create_indexes(neo4j_session):
     # ensure the idempotency of creating indexes
     create_indexes.run(neo4j_session, None)
-    create_indexes.run(neo4j_session, None)
+    create_indexes.run(neo4j_session, None)
```

Comment:
Removing this empty line will cause the linter to complain; add it back to make it stop complaining